cmd/k8s-operator/deploy/crds/tailscale.com_connectors.yaml

```diff
@@ -40,38 +44,39 @@ spec:
             metadata:
               type: object
             spec:
-              description: Desired state of the Connector resource.
+              description: ConnectorSpec describes the desired Tailscale component.
               type: object
-              required:
-                - subnetRouter
               properties:
+                exitNode:
+                  description: ExitNode defines whether the Connector node should act as a Tailscale exit node. Defaults to false. https://tailscale.com/kb/1103/exit-nodes
+                  type: boolean
+                hostname:
```

Comment:
in this PR hostname has moved `.spec.subnetRouter.hostname` -> `.spec.hostname`, this corresponds to the API change where the one `Connector` instance corresponds to one Tailnet node that can however be both an exit node and a subnet router